front/lib/resources/group_resource.ts

```diff
@@ -548,6 +566,26 @@ export class GroupResource extends BaseResource<GroupModel> {
     return groups.map((group) => new this(GroupModel, group.get()));
   }
 
+  async isMember(auth: Authenticator): Promise<boolean> {
+    const owner = auth.getNonNullableWorkspace();
+
+    if (this.isGlobal()) {
```

Comment:
For my own curiosity, can user be member of `system` agent?

Reply:
you mean group ?
everybody is member of global group, nobody of system group

Reply:
Yes indeed I meant group. Ok get it, and if the group is kind `system` and nobody can be part of a system group, does going to the query below is needed or could we just skip and return `false`?